docker-bake.hcl

```diff
@@ -72,6 +74,14 @@ function "magistrate_code_coverage" {
   result = lookup(item, "magistrate_code_coverage", "0")
 }
 
+function "magistrate_suffixes" {
+  params = [item]
+  result = join("", compact([
+    magistrate_build_against_vt(item) == 1 ? "-vt" : "",
+    magistrate_docs(item) == 1 ? "-docs" : ""
+  ]))
+}
+
```

Comment:
Can you check the logic for `variant` in workflows `docker-bake.hcl`? I'd prefer something closer to that for consistency and having the suffix right in the build definition.

Reply:
(I'll be adding something similar in `vt` as well)